Fix instructions on contributing file

Under `I have found a bug!` section, we're listing two invalid commands on step 1. Git pull do not have the switch `-u` and there's no registered command `package` on `package.json`. Instead, we should use `git pull` and `npm run pack`.

```bash
$: git pull -u
error: unknown switch `u'
usage: git pull [<options>] [<repository> [<refspec>...]]

    -v, --verbose         be more verbose
    -q, --quiet           be more quiet
    --progress            force progress reporting

Options related to merging
    -r, --rebase[=<false|true|preserve|interactive>]
                          incorporate changes by rebasing rather than merging
    -n                    do not show a diffstat at the end of the merge
    --stat                show a diffstat at the end of the merge
    --log[=<n>]           add (at most <n>) entries from shortlog to merge commit message
    --squash              create a single commit instead of doing a merge
    --commit              perform a commit if the merge succeeds (default)
    --edit                edit message before committing
    --ff                  allow fast-forward
    --ff-only             abort if fast-forward is not possible
    --verify-signatures   verify that the named commit has a valid GPG signature
    -s, --strategy <strategy>
                          merge strategy to use
    -X, --strategy-option <option=value>
                          option for selected merge strategy
    -S, --gpg-sign[=<key-id>]
                          GPG sign commit

Options related to fetching
    --all                 fetch from all remotes
    -a, --append          append to .git/FETCH_HEAD instead of overwriting
    --upload-pack <path>  path to upload pack on remote end
    -f, --force           force overwrite of local branch
    -t, --tags            fetch all tags and associated objects
    -p, --prune           prune remote-tracking branches no longer on remote
    --recurse-submodules[=<on-demand>]
                          control recursive fetching of submodules
    -j, --jobs[=<n>]      number of submodules pulled in parallel
    --dry-run             dry run
    -k, --keep            keep downloaded pack
    --depth <depth>       deepen history of shallow clone
    --unshallow           convert to a complete repository
    --update-shallow      accept refs that update .git/shallow
    --refmap <refmap>     specify fetch refmap
```

```bash
$: npm run package
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/lbalves/.nvm/versions/node/v4.4.7/bin/node" "/Users/lbalves/.nvm/versions/node/v4.4.7/bin/npm" "run" "package"
npm ERR! node v4.4.7
npm ERR! npm  v3.10.5

npm ERR! missing script: package
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lbalves/Sites/black-screen-clone/npm-debug.log
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,15 +1,15 @@
 I have found a bug!
 -------------------
 
-Awesome, but before you [report it to us](issues/new), make sure to [check whether this has already been reported](issues?q=is%3Aissue). 
+Awesome, but before you [report it to us](issues/new), make sure to [check whether this has already been reported](issues?q=is%3Aissue).
 If not, before reporting the issue you'll need to gather some information by following these instructions:
 
 1. Make sure you are using the latest version of the application:
 
   ```bash
-  git pull -u
+  git pull
   rm -r "/Applications/Black Screen"*
-  npm run package
+  npm run pack
   ```
 2. If the bug is still present, [open an issue](issues/new).
 3. Write steps to reproduce the bug.
@@ -17,11 +17,11 @@ If not, before reporting the issue you'll need to gather some information by fol
 5. Gather debug logs.
 
  5.1. Open developer tools (View -> Toggle Developer Tools).
- 
+
  5.2. Find Console.
- 
+
  5.3. Copy the output and paste it into the issue.
- 
+
 I have some important changes!
 ------------------------------
 

